vscode/src/extension.ts

```diff
@@ -77,45 +75,34 @@ async function missingJagSetupPrompt(jagExec: string) : Promise<boolean> {
 
 }
 
-async function invalidCLIVersionPrompt(toitExec: string, version?: string | null): Promise<void> {
-  const updateToitAction = "Update toit executable";
-  const action = await Window.showErrorMessage(`Toit executable${version ? ` ${version}` : ""} is outdated. Please update the executable to use the extension (reload window to activate the extension).`, updateToitAction);
-  if (action === updateToitAction) {
-    cp.execFileSync(toitExec, ["update"]);
-  }
-}
-
-async function missingCLIPrompt(path: string|null) {
-  const installAction = "Install";
-  const settingsAction = "Update settings";
-  let message = "Could not find the `toit` executable";
-  if (path !== null) {
-    message += " at '" + path + "'";
-  }
-  message += ". Please make sure `toit` is installed and set the toit.path setting to the executable (reload the window to activate the extension";
-  const action = await Window.showErrorMessage(message, installAction, settingsAction);
-  if (action === installAction) {
-    env.openExternal(Uri.parse("https://docs.toit.io/getstarted/installation"));
-  } else if (action === settingsAction) {
-    Commands.executeCommand( "workbench.action.openSettings", "toit.path" );
+async function missingLspPrompt() {
+  const installJagAction = "Install Jaguar";
+  let message = "Could not find the `jag` or `toit` executable. ";
+  message += "Please make sure `jag` or `toit` is installed or set the jag.path ";
```

Comment:
```suggestion
  message += "Please make sure `jag` or `toit` is installed or let the jag.path ";
```

Reply:
Changed to:
"Please make sure `jag` or `toit` is installed. If not in PATH, update the 'jag.path' (or 'toit.path') setting. Then reload the window."